website/src/components/DeploymentTypeLabel/index.jsx

```diff
@@ -0,0 +1,16 @@
+import React from "react";
+
+export const DeploymentTypeLabel = ({deploymentType, remarks}) => {
+
+  return (
+  <div>
+    <div className="deploymentTypeContainer">
+      {deploymentType.map((type) => {
+          if (type === "DNS") return <div className="deploymentTypeDNS">DNS</div>
+          if (type === "DID") return <div className="deploymentTypeDID">DID</div>
```

Comment:
non-blocking: Do we need to return some display if `type` happens to be something other than DNS and DID? This could work too since it's only for the docs and we are certain that `type` will always be either DNS or DID only.

Reply:
i dont think we need cos for deployment type for now there are only 2 types, DNS and DID so it doesn't make sense to have something other than DNS and DID. 

if not, i'll make the default DNS? smt like:
```type === "DID" ?  {DID} : {DNS}```

Reply:
Oh I see. Got it, then it's probably not necessary if there are only 2 types.👍

> if not, i'll make the default DNS? smt like:
> `type === "DID" ? {DID} : {DNS}`

This one might have the risk of accidentally assuming as DNS in the docs if we accidentally provided an invalid type. At least now it's just empty. haha

Reply:
ahh ok! sure thing!